### DISCUSSION TITLE
4 Real buttons + IR module

### DISCUSSION THREAD

**Discussion by gaboskar001** [HomeSpan/HomeSpan#201] [Q&A]
Hey guys!
I would like to do a project with 4 real buttons + IR module. Can someone help me? My code already works with the home app. The problem is that the status is just not updated in the app. I started with the project '05WorkingLed'.
thanks in advance!

**Comment by HomeSpan** [comment]
See [Example 15](https://github.com/HomeSpan/HomeSpan/blob/master/examples/15-RealPushButtons) - it shows you how to implement real buttons and update status.
